### DISCUSSION TITLE
Can you `include` files from a parent directory?

### DISCUSSION THREAD

**Discussion by aromanielloNTIA** [quarto-dev/quarto-cli#9956] [Q&A]
### Description

Hi there, I'm working on building a site with Quarto and I am wondering if the following is possible. Given a project structure such as:

```
- _quarto.yml
- index.qmd
- includes/
    - snippet_a.qmd 
- some_folder/
    - page_a.qmd
    - snippet_b.qmd
- other_folder/
    - page_b.qmd
``` 

I'd like to use the [include shortcode](https://quarto.org/docs/authoring/includes.html) to include content from `includes/snippet_a.qmd` in pages generated by `page_a.qmd` and `page_b.qmd`. Unfortunately, it seems like I can't use the typical syntax to specify a file path relative to the project root in the include shortcode. For example, this doesn't work in `page_a.qmd`:

```
{{< include /includes/snippet_a.qmd >}}
```

However, it does work if I use a local/relative path, such as:

```
{{< include snippet_b.qmd >}}
```

Is there another way to include files from another directory? Using parent directory syntax like `../something/` works but does not work beyond traversing a single level (i.e., `../../something` does not resolve).

Any help is appreciated!

**Comment by mcanouil** [comment]
You cannot include a parent directly of a Quarto project by design.
A Quarto project is basically defined by `_quarto.yml`.

Inside a Quarto project a leading `/` means at the root of the project. So you can use that to include anything from the project's root.
Of course, you can also use relative path (relative to the document calling the include shortcode).

If it does not work, please share a small reproducible example and the output of `quarto check`.

**Reply by aromanielloNTIA** [reply]
Thanks for your reply! I am aware that files outside of the project cannot be included- I am not trying to do that. See the project structure I shared in my original question. However, I think I solved this one- I am using the VS Code Quarto extension. When using the `include` from root-of-project, I had to generate the preview from the project root, rather than just at the sub-page. This is working as expected for me now (i.e., `{{< include /some/file/in/project.qmd >}}` now is included correctly).

**Comment by aromanielloNTIA** [comment]
Perhaps this is a bug in the VS Code extension? The following fails:

`quarto preview some_folder/page_a.qmd`

While the following succeeds:

`quarto preview index.qmd`
